upstairs/src/lib.rs

```diff
@@ -5498,6 +5550,10 @@ impl BlockOp {
             BlockOp::Write { offset: _, data } => {
                 Some(ceiling_div!(data.len(), iop_sz))
             }
+            BlockOp::WriteUnwritten { offset: _, data } => {
+                // XXX maybe don't for the scrubber?
```

Comment:
Probably not - this type of non-Guest traffic shouldn't count towards any of their disk limits, otherwise the background stuff we're doing competes with their real IO.

Also, we want the scrub to complete as quickly as possible :)

Reply:
I think for now, yeah.

We may reconsider this at some point.  If the user has a bunch of volumes and they are all scrubbing like crazy, we could possibly swamp the network?  Maybe?  I'll let it become an actual problem before I try to solve it though.